tests/ViewTests.elm

```diff
@@ -48,4 +60,14 @@ all =
                     |> TestContext.simulate (Query.find [ Selector.tag "button" ]) Events.Click
                     |> TestContext.expectView
                     |> Query.has [ Selector.tag "p" ]
+        , test "fails when triggering events on a not found element" <|
+            \() ->
+                htmlProgram
+                    |> TestContext.simulate (Query.find [ Selector.tag "foo" ]) Events.Click
+                    |> expectError "Query.find always expects to find 1 element, but it found 0 instead."
+        , test "fails when triggersingevents on an element that does not handle that event" <|
+            \() ->
+                htmlProgram
+                    |> TestContext.simulate (Query.find [ Selector.tag "button" ]) Events.DoubleClick
+                    |> expectError "Failed to decode string"
```

Comment:
Yes, this is due to this line: https://github.com/rogeriochaves/elm-html-test/blame/master/src/Test/Html/Events.elm#L123

I don't know why it is there, I'll make a PR to remove it, the error message is much better without it.